.editorconfig

```diff
@@ -0,0 +1,13 @@
+[*]
+indent_style = space
+indent_size = 4
+binary_next_line   = true
+
+[Makefile]
+indent_style = tab
```

Comment:
Maybe this is not needed because `shfmt` does not do anything to Makefiles?
```suggestion
```

Reply:
This might be a lit misleading. But there are two different lists of files touched depending the arguments. Specifically `shfmt -l .` vs `shfmt -l $(git ls-files)`. Despite the former doesnt seem to touch Makefile, i though that it doesnt hurt to leave it in the `.editorconfig` and avoid any mistake.

Reply:
When you pass filenames to shfmt, it will try to format everything it can, even yaml files. So that should be used only if you want to format a certain file.

Reply:
In that case we just need to be careful to not execute it on yaml files (or add a definition which will be in sync with the yamllint, but that is probably not ideal). I am OK with that.

Reply:
I guess my answer was misleading.
shfmt cannot understand and format YAML. It formats it, but then it will result in something different, likely even invalid YAML.
For formatting YAML I will introduce yamltidy config files (already did that in os-autoinst).

Reply:
In that case we are back in the original question, can `shfmt` format makefiles (and by format I mean will the makefile still be a valid makefile)? If not, we should definitely drop this section from the configuration to avoid any confusion. The documentation of that tool does not suggest it understands anything besides shell scripts.

Reply:
Yes, I think it should be dropped, if it can't format Makefiles correctly